auto_submit/test/requests/github_webhook_test.dart

```diff
@@ -2,39 +2,43 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:convert';
-
 import 'package:auto_submit/requests/github_webhook.dart';
-import 'package:github/github.dart';
 import 'package:shelf/shelf.dart';
 import 'package:test/test.dart';
 
-import '../src/service/fake_config.dart';
 import './github_webhook_test_data.dart';
+import '../src/service/fake_config.dart';
+import '../src/service/fake_github_service.dart';
 
 void main() {
   group('Check Webhook', () {
     late Request req;
     late GithubWebhook githubWebhook;
+    late FakeConfig config;
+    late FakeGithubService githubService;
+    late Future<FakeGithubService> futureGithubService;
 
     setUp(() {
       req = Request('POST', Uri.parse('http://localhost/'),
           headers: {
             'header1': 'header value1',
           },
           body: webhookEventMock);
-      githubWebhook = GithubWebhook(
-        config: FakeConfig(),
-      );
+      githubService = FakeGithubService();
+
+      Future<FakeGithubService> getFuture(FakeGithubService githubService) async {
+        return githubService;
+      }
+
+      futureGithubService = getFuture(githubService);
```

Comment:
Seems not needed.

Reply:
Hi Keyong, since the method createGithubService() in fakeConfig class only returns future data type, here I think it is necessary to create a Future<FakeGithubService> object to construct the fakeconfig. 
I searched and found an easier way to create Future<T> from T: futureGithubService = Future.value(githubService).